### ISSUE TITLE
Recipients do not receive notifications when a report is opened

### ISSUE THREAD

**Issue by Hooteo** [globaleaks/GlobaLeaks#3684]
### What version of GlobaLeaks are you using?

4.13.12

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows, macOS, iOS

### Describe the issue

I am using version 4.13.12. I set up a Globaleaks server for the company to report problems.
I configured the SMTP with google (to keep everything external and off the LAN) and the test email arrives correctly.
I read on the Globaleaks threat model (https://docs.globaleaks.org/en/main/security/ThreatModel.html) that:
Email notifications -> Data sent to notify recipients of a new report via email.
So I expect that when a colleague creates a report the recipient will be notified so they can see it.
However, this does not happen and I cannot understand why.
Nothing has been disabled on the notifications.
Is it possible to implement this or figure out how to make it work?

Thanks,

**Comment by evilaliv3** [comment]
@Hooteo : Does it work if you restore the default SMTP configuration?

**Comment by Patryz** [comment]
Just letting you know, we've had this  issue as well for a few versions, it seems to appear randomly. Restarting the globaleaks service sends all of the notifications at once.

We are on 4.13.8, using external smarthost for sending.

**Comment by Patryz** [comment]
It's totally random for me, so far I haven't been able to figure out a cause.

Scheduled a restart of the service in the night, hopefully it should mitigate the issue until a fix is in place.

**Comment by Hooteo** [comment]
@Patryz Yes i have that configuration in my company. The machine is an Ubuntu 22.04 in DMZ updated with only globaleaks installation.
I'm gonna add a chrono job to run systemctl restart globaleaks every night.
@evilaliv3 do you think it's possible to implement a flush of the emails every N hours?

**Comment by flashlight4** [comment]
Same issue with me. eMail-Notification works only with standard mail server from GL. With custum configuration with working and otherwise tested mail server: no notifications were sent.

Restarting the server (service globaleaks restart) shows no effect.

Configuration:
Ubuntu 22.04
Cloud Server at Hetzner
Version GL: 4.13.12

OS Clients: MacOS, Windows, several Browsers

**Comment by evilaliv3** [comment]
@flashlight4: many could be the reason; here follow some

- if you have enabled anonymization of outgoing connections, get sure your mail server has a public IP and is accepting Tor connections
- if your mail server is a remote one ensure it has a valid tls certificate and if it is a self signed certificate ensure to add it to the local trust store of the globaleaks server

**Comment by flashlight4** [comment]
@evilaliv3 Thx for Your quick response. 
- The anomyization of outgoing connections is not enabled. The site is reachable via TOR.
- My mail server is remote at the provider Strato (SMTP-Server: smtp.strato.de, Port: 465, Authorization with correct credentials is ensured). According to SSL Labs-Test the certificates should be valid. https://www.ssllabs.com/ssltest/analyze.html?d=smtp.strato.de 
- If helpful I could provide the protocols as well.
